README.md

```diff
@@ -56,6 +56,21 @@ Added in 1.4.0. If you have run the `add-index` command and have verified that t
 
 Added in 1.4.0. If you have run the `add-index` command and something unexpected has occurred, just run the `primary-key-revert` command and the backup table will immediately be returned to being the active table.  
 
+### WordPress Multisite
+As of 1.4.1 the `add-index`, `primary-key-add` and `primary-key-revert` commands only apply to a single site. This means that to run on a WordPress multisite, for sites beyond the main site, you would need to pass the `--url=` flag for each subsite.
+
+However, you can script this process in bash by getting a list of sites and looping over them:
+
+```bash
+for site in $(wp site list --field=url);
+then
+	wp pantheon session add-index
```

Comment:
Should this command use the site variable?

Reply:
```suggestion
	wp pantheon session add-index --url="$site"
```

Reply:
yerp it should. I'll need to open a new PR to make sure this gets merged to develop.

Reply:
oh wait also it should be `for ... do ... done` not `then`

Reply:
fixed in #273